app/src/components/RobotSettings/ConfirmStartDeckCalModal.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from 'react'
+
+import { AlertModal } from '@opentrons/components'
+import styles from './styles.css'
+
+export type ConfirmStartDeckCalModalProps = {|
+  cancel: () => mixed,
+  confirm: () => mixed,
+|}
+
+const HEADING = 'Are you sure you want to continue?'
+const CANCEL = 'cancel'
+const EXIT = 'continue'
+const WARNING =
+  'Performing a deck calibration will clear your instrument offset calibrations.'
```

Comment:
We should probably get ahead of the game 
```suggestion
  'Performing a deck calibration will clear your pipette offset calibrations.'
```